rfcs/97-add-maximum-file-size-for-storage-write.md

```diff
@@ -0,0 +1,71 @@
+---
+author: JinnyYi <github.com/JinnyYi>
+status: draft
+updated_at: 2021-06-09
+---
+
+# GSP-97: Add Maximum File Size for Storage Write
+
+## Background
+
+Depending on the size of the data you are uploading, services offer different options, like upload a file in a single operation, in parts, or through append operation.
+
+We use `Write` to handle single write operation. And the size is not unlimited.
+
+- S3 alike storage services have 5GB limit for a single PUT operation.
+- azblob limited to 5000 MiB <https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob>
+
+If we upload a file with size out of limit in a single operation, we will get an error like `Request Entity Too Large` in azblob. Same with `copy`,`fetch`, etc.
+
+We should figure out the error before sending request to reduce the consumption of network resources.
+
```

Comment:
Please add more context about `StorageMeta`.

- What's our current design?
- What storage meta we provide?